clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -148,6 +160,24 @@ class AnnotatingParser {
     }
   }
 
+  const FormatStyle::FunctionDeclarationWithKeywords *
+  isInsideFunctionWithKeywordedParameters(const FormatToken &Token) const {
```

Comment:
The name is misleading, I'd expect a `bool`ean return value with this name.

Reply:
`findSurroundingFunctionWithKeywordedParameters()`?